src/components/v2v3/V2V3Project/V2V3ProjectSettings/pages/ProjectNftSettingsPage/ProjectNftSettingsPage.tsx

```diff
@@ -0,0 +1,276 @@
+import { t, Trans } from '@lingui/macro'
+import { useForm } from 'antd/lib/form/Form'
+import { Button, Form } from 'antd'
+import { JuiceInput } from 'components/inputs/JuiceTextInput'
+import { ProjectMetadataContext } from 'contexts/shared/ProjectMetadataContext'
+import { V2V3ProjectContext } from 'contexts/v2v3/Project/V2V3ProjectContext'
+import { useSetThemeTx } from 'hooks/v2v3/transactor/SetThemeTx'
+import { CSSProperties, useContext, useState } from 'react'
+import styles from './Capsules.module.css'
+import { Rule } from 'antd/lib/form'
+import { CustomResolverForm } from './CustomResolverForm'
+import { helpPagePath } from 'utils/routes'
+import ExternalLink from 'components/ExternalLink'
+
+export function ProjectNftSettingsPage() {
+  const { handle } = useContext(V2V3ProjectContext)
+  const { projectId } = useContext(ProjectMetadataContext)
+  const [themeForm] = useForm()
+  const [loadingSetTheme, setLoadingSetTheme] = useState<boolean>()
+
+  const [previewTextColor, setPreviewTextColor] = useState<string>('FF9213')
+  const [previewBgColor, setPreviewBgColor] = useState<string>('44190F')
+  const [previewAltBgColor, setPreviewAltBgColor] = useState<string>('3A0F0C')
+
+  const setThemeTx = useSetThemeTx()
+
+  function onSetThemeFormSaved() {
+    setLoadingSetTheme(true)
+
+    const colors = {
+      textColor: String(themeForm.getFieldValue('textColor'))
+        .trim()
+        .toUpperCase(),
+      bgColor: String(themeForm.getFieldValue('bgColor')).trim().toUpperCase(),
+      altBgColor: String(themeForm.getFieldValue('altBgColor'))
+        .trim()
+        .toUpperCase(),
+    }
+
+    setThemeTx(colors, {
+      onDone: () => setLoadingSetTheme(false),
+    })
+  }
+
+  function updatePreview() {
+    setPreviewTextColor(
+      String(themeForm.getFieldValue('textColor')).trim().toUpperCase(),
+    )
+    setPreviewBgColor(
+      String(themeForm.getFieldValue('bgColor')).trim().toUpperCase(),
+    )
+    setPreviewAltBgColor(
+      String(themeForm.getFieldValue('altBgColor')).trim().toUpperCase(),
+    )
+  }
+
+  const colorInputRules: Rule[] = [
+    {
+      pattern: new RegExp('^[0-9a-fA-F]{6}$'),
+      message: t`Enter a valid color hex code.`,
+    },
+    { required: true, message: t`Each field is required.` },
+  ]
+
+  const textStyle: CSSProperties = {
+    fill: '#' + previewTextColor,
+    whiteSpace: 'pre',
+    fontSize: '16px',
+    fontWeight: 500,
+    fontFamily: '"Capsules-500", monospace',
+  }
```

Comment:
**suggestion** could we tailwind these?

Reply:
I would need to add Capsules to the tailwind config, and I'm not sure if it makes sense to tailwind the `fill` property (because it does change on user input).

The others would be `"whitespace-pre text-base font-medium"`. I think it makes sense to keep everything as a `CSSProperties`, but if you'd like I can break those out and apply both.